tests/foreman/api/test_notifications.py

```diff
@@ -362,3 +392,43 @@ def test_positive_notification_recipients(target_sat):
     recipients = target_sat.api.NotificationRecipients().read()
     for notification in recipients.notifications:
         assert set(notification_keys) == set(notification.keys())
+
+
+@pytest.mark.tier3
+@pytest.mark.parametrize(
+    'admin_user_with_custom_settings',
+    [
+        pytest.param({'disabled': True, 'mail_enabled': True}, id='account_disabled'),
+        pytest.param({'disabled': False, 'mail_enabled': False}, id='mail_disabled'),
+    ],
+    indirect=True,
+)
+@pytest.mark.usefixtures(
+    'reschedule_long_running_tasks_notification',
+    'wait_for_no_long_running_task_mail',
+)
+def test_negative_no_notification_for_long_running_tasks(
+    admin_user_with_custom_settings, long_running_task, root_mailbox_copy
```

Comment:
The code looks rock-solid. One question here, why does this test the `root_mailbox_copy` fixture?

Reply:
Good catch, I actually wanted to ensure that a clean mailbox file is created, but it is already done in fixture `wait_for_no_long_running_task_mail` via `clean_root_mailbox`. 
But since you mentioned it, I decided to keep it and added few lines to check for the unexpected mail explicitly. 
Read more below...